packages/SwingSet/test/test-netstring.js

```diff
@@ -102,12 +102,13 @@ test('decode', t => {
   eq(expectedBuffer, [emoji], '');
 
   function bad(input, message) {
-    t.throws(() => decode(Buffer.from(input)), { message });
+    t.throws(() => decode(Buffer.from(input), 100), { message });
   }
 
   // bad('a', 'non-numeric length prefix');
   bad('a:', /unparseable size .*, should be integer/);
   bad('1:ab', 'malformed netstring: not terminated by comma');
+  bad('101:x', /size .* exceeds limit of 100/);
```

Comment:
please test both edges of the limit, so the test is as tight as the specification, e.g. a passing test that decides a 100-byte payload